docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.montagu.dide.ic.ac.uk:5000/orderly.server:f427544
+FROM docker.montagu.dide.ic.ac.uk:5000/orderly.server:master
```

Comment:
I've set this up to point at master which should track changes a little better as upstream changes